src/app/listen/listen.js

```diff
@@ -444,23 +450,28 @@ angular
 
                         if (linkType === "previous") {
                             var lowerBound = ($routeParams.start - stepBy);
+                            var uriPrev;
+
+                            // no previous link if we are at the start
+                            if ($routeParams.start <= 0) {
+                                uriPrev = "#";
```

Comment:
why return `#` here instead of just an empty string?

We'd want the button disabled, no?

Reply:
I think I did that to minimise the changes to the listen controller, as it was already returning '#' for unavailable audio. I have changed it to return empty string now.